docs/user_needs/zephyr_rtos_user.sdoc

```diff
@@ -0,0 +1,23 @@
+[DOCUMENT]
+TITLE: Zephyr RTOS User
+REQ_PREFIX: ZEP-
+
+[GRAMMAR]
+IMPORT_FROM_FILE: user_needs.sgra
+
+[FREETEXT]
+
+TBD
+
+[/FREETEXT]
+
+[USER_NEED]
+STATUS: Draft
+TYPE: User Need
+USER: Zephyr RTOS User
+TITLE: Easily Switch Application to different MCU Architecture
+STATEMENT: >>>
```

Comment:
This is a perfect example of a very valid User Need that fits in this fragment. Could you add two more good examples to demonstrate your proposal even stronger?

Reply:
See above.

Reply:
I have removed the Software Developer section from this PR and added a handful of additional User Requirements from a Product Manufacturer. Justifications can be added if needed.